### DISCUSSION TITLE
Links on wiki home page have preceding 'wiki/' that breaks them if the URL has a trailing slash

### DISCUSSION THREAD

**Discussion by chuckwagoncomputing** [community/community#7157] [Other Feature Feedback, Questions, & Ideas]
Here is an example; try clicking the link in the page:
https://github.com/chuckwagoncomputing/ideal-tribble/wiki/

If you remove the trailing slash from the above URL, then it works.

**Comment by rusefillc** [comment]
OMG https://github.com/chuckwagoncomputing/ideal-tribble/wiki/ and https://github.com/chuckwagoncomputing/ideal-tribble/wiki do not work the same?! that's unexpected :(
